### DISCUSSION TITLE
Extend organization data model

### DISCUSSION THREAD

**Discussion by ddeboer** [netwerk-digitaal-erfgoed/solid-crs#397] [Collections]
For the publication module (#375) we need some more organization data. So extend the current data model (https://github.com/netwerk-digitaal-erfgoed/solid-crs/discussions/162) with:

- `schema:description` for the about box and page;
- - `schema:logo` for branding;
- `schema:contactPoint` so visitors know how to contact the organization;
- `schema:sameAs` to link to the organization’s website if different than the resource’s IRI

We then end up with:

```ttl
@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
@prefix schema: <http://schema.org/> .

<https://data.hetlageland.org/>
    rdf:type schema:Organization ;
    schema:name "Heemkundevereniging Het Lage Land" ;
    schema:alternateName "Het Lage Land" ; # Optional
    schema:description "Het Lage Land specialiseert zich in alles wat te maken heeft met Het Lage Land. Hier kan nog een langer verhaaltje komen."; # Optional
    schema:logo <https://www.hetlageland.org/logo.svg> ; # Optional
    schema:contactPoint [a schema:ContactPoint ; schema:email "collecties@hetlageland.org"] ; # Optional
    schema:sameAs <https://www.hetlageland.org> . # Optional
```

Should this data be edited directly in the pod for now?

@sdevalk Does this make sense from the perspective of the Basisregistratie or are we out of its scope here anyway?

**Comment by sdevalk** [comment]
@ddeboer 

> Does this make sense from the perspective of the Basisregistratie or are we out of its scope here anyway?

Yes, this makes sense! Indeed, this is out of scope of the Basisregistratie - that merely requires an organization's identifier and name. So we can extend it safely.

> schema:sameAs <https://www.hetlageland.org> . # Optional

Or we could use `schema:url`?
